lms/djangoapps/discussion/views.py

```diff
@@ -718,20 +718,21 @@ def _discussions_mfe_context(query_params: Dict, course_key: CourseKey, user: Us
     mfe_url = get_discussions_mfe_url(course_key)
     if not mfe_url:
         return {"show_banner": False, "show_mfe": False}
-    show_banner = bool(has_access(user, 'staff', course_key))
+    discussions_mfe_enabled = ENABLE_DISCUSSIONS_MFE.is_enabled(course_key)
```

Comment:
Do we need to pass course_key here?

Reply:
Yes, If in future we want to turn off new mfe for the specific course we can override for that course.